internal/testutil/testutil.go

```diff
@@ -89,11 +90,22 @@ func testContext() (Context, error) {
 		return tc, noProjectID
 	}
 
-	pkgs, err := packages.Load(&packages.Config{Mode: packages.NeedName}, "github.com/GoogleCloudPlatform/golang-samples")
+	cfg := &packages.Config{
+		Mode:  packages.NeedName | packages.NeedFiles,
+		Tests: true,
+	}
+	pkgs, err := packages.Load(cfg, "github.com/GoogleCloudPlatform/golang-samples")
 	if err != nil {
-		return tc, fmt.Errorf("Could not find golang-samples: %v", err)
+		return tc, fmt.Errorf("could not find golang-samples: %v", err)
+	}
+	for _, pkg := range pkgs {
+		if len(pkg.GoFiles) > 0 && strings.HasSuffix(pkg.GoFiles[0], ".go") {
+			tc.Dir = filepath.Dir(pkg.GoFiles[0])
```

Comment:
So the problem with the current code is that instead of getting the path of the golang-samples directory we need the path to the first file? I'm not clear why this is needed, a code comment would be useful for future reference.

Reply:
The file is in the package, which is in the root of the repo. So, if we get the `Dir` of that file, it will be the directory of the package. I hope that makes sense.

Reply:
But why is that different from the PkgPath?

Reply:
`PkgPath` is the Go package path (`github.com/GoogleCloudPlatform/golang-samples`). This is getting the file path (`/home/.../golang-samples`).